### ISSUE TITLE
Redirecting Sensei archive

### ISSUE THREAD

**Issue by justintadlock** [lightspeeddevelopment/lsx#167]
This code in `inc/sensei.php` probably doesn't belong in the theme:

```
function lsx_sensei_redirect_to_home( $query ){
	if ( ! is_admin() && is_post_type_archive( 'lesson' ) && $query->is_main_query() ) {
		wp_redirect( home_url() . '/courses-overview' );
		exit;
	}
}
add_action( 'parse_query', 'lsx_sensei_redirect_to_home' );
```

There's a few issues here, so I'll just cover them one at a time.

**home_url()**

First, wrap this in `esc_url_raw()` for redirects.

**Correct use of home_url()**

It should look like `home_url( 'courses-overview' )`.

**Correct URL?**

Is it OK to assume that `sitehome/courses-overview` is the absolute correct URL?  I haven't tested Sensei, but I'd assume that any permalink could be very different.

**Redirecting**

Generally speaking, themes should not be redirecting anything.  That's something best left to core and plugins.

**Comment by ashleyshaw** [comment]
@justintadlock our plan was to make the LSX theme compatible with the Woo Sensei LMS plugin, how would you add support to the theme if you didn't add the include file? Thx

**Comment by justintadlock** [comment]
I think there's a misunderstanding.  I'm talking about redirecting the archive, not including the file.

**Comment by nandotess** [comment]
Resolved on LSX 2.0.0 branch.
